### DISCUSSION TITLE
編集していない場合は下書きを保存しないでほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#16] [要望・アイデア・不具合]
変更していない記事が「編集中」として下書きにどんどん溜まってしまいます。
手動で削除するのが手間なので、変更がない場合は下書きに保存しないでほしいです。

**Comment by kskwtnk** [comment]
@uhooi 

ご意見ありがとうございます！

運営としてこんなことを言うのもどうなのか……となりますが、こちら非常に分かります……😭
こちらは既にIssueとしては存在していて、対応はします！

**Reply by uhooi** [reply]
ですよねーｗ
おお、すでに Issue が…ありがとうございます！

**Comment by blono** [comment]
私は手動で削除するのは諦めてますね。
これIssueとしてあるということは、この機能を実装後に、現状編集中だけど内容が編集されていない多くの下書き達の扱いどうするの？みたいな議論とかあるあるですよね。頑張ってください！（丸投げ💪

**Reply by kskwtnk** [reply]
@blono 

> この機能を実装後に、現状編集中だけど内容が編集されていない多くの下書き達の扱いどうするの？みたいな議論とかあるあるですよね。

そうなんですよ……そこも難しそうでして、すぐに解決できるイメージはあまり持てていないのが正直なとこです…… 😭
